storage/range_command.go

```diff
@@ -1053,7 +1053,7 @@ func (r *Range) mergeTrigger(batch engine.Engine, merge *proto.MergeTrigger) err
 
 	if !r.Desc().EndKey.Less(merge.UpdatedDesc.EndKey) {
 		return util.Errorf("range end key is not less than the post merge end key: %s < %s",
-			r.Desc().EndKey, merge.UpdatedDesc.StartKey)
+			r.Desc().EndKey, merge.UpdatedDesc.EndKey)
```

Comment:
Is this meant to be
merge.UpdatedDesc.EndKey, r.Desc().EndKey)
to match the error message?